test/mocks/data/firestore_user_comment.dart

```diff
@@ -0,0 +1,48 @@
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/database/user/user_id_firestore.dart";
+import "package:proxima/models/database/user_comment/user_comment_data.dart";
+import "package:proxima/models/database/user_comment/user_comment_firestore.dart";
+import "package:proxima/models/database/user_comment/user_comment_id_firestore.dart";
+import "package:proxima/services/database/user_comment_repository_service.dart";
+
+import "user_comment_data.dart";
+
+class UserCommentFirestoreGenerator {
+  int _userCommentId = 0;
+  final UserCommentDataGenerator _userCommentDataGenerator;
+
+  UserCommentFirestoreGenerator({int seed = 0})
+      : _userCommentDataGenerator = UserCommentDataGenerator(seed: seed);
+
+  Future<List<UserCommentFirestore>> addComments(
```

Comment:
I'm feeling like the two functions in this class have a very close meaning; the second one is almost `addComments(1, ..., ...)`. Why do you need the two? Couldn't they be merged?
What do you think?

Reply:
I actually need the second one (`createMockUserComment`) when I have to get a `UserCommentFirestore` but doesn't need to have it in the database.

This happens in `user_comment_firestore_test.dart` for instance.
Merging the two and keeping only `addComments`, I would need to create a repository and add the comments to the database to get the mock data. And I don't think it makes a lot of sense for this test.

Thus, I personnaly prefer to keep the two functions here if that's ok for you ?

Reply:
Note: Those function have slightly modified after removing the `UserCommentIdFirestore`.

I actually need the second one (`createMockUserComment`) when I have to get a `UserCommentFirestore` but doesn't need to have it in the database.

This happens in `user_comment_firestore_test.dart` for instance.
Merging the two and keeping only `addComments`, I would need to create a repository and add the comments to the database to get the mock data. And I don't think it makes a lot of sense for this test.

Thus, I personnaly prefer to keep the two functions here if that's ok for you ?

Reply:
What you say make sense. I like the fact that you use the second function explicitely in the first one now, to decrease code repetition. Good job :)